src/components/Reports/TeamReport/components/ReportCharts.css

```diff
@@ -84,4 +84,8 @@
     height: 16px;
     border-radius: 4px;
     margin-right: 8px;
+}
+
+.dark .team-report-chart-teams {
+    background-color: #333;

```

Comment:
``background-color: #333`` appears 16 times. Maybe we can condense that or link them all to a variable so it's not hard-coded.